### ISSUE TITLE
Compare stats between two time periods

### ISSUE THREAD

**Issue by adriaanvanrossum** [simpleanalytics/roadmap#13]
Thanks to @lseelenbinder on Twitter

**Comment by khrome83** [comment]
It would be great if this just compared the previous period by default. I love how simple - simple analytics is. Entering random dates does not appeal to me. But if I am viewing the current month, it would be great to know if I am trending up or down. I don't think layering month A overtop of month B makes sense, because of the weekends and such fall on different days. What I need to know is am I trend up down or up compared to the preview time window, and by what percent. Raw data would be ok also, 34000 pageviews in the previous window vs 23000 in the current for example.

**Comment by adriaanvanrossum** [comment]
Good point, just a percentage change behind the numbers would be okay, right?

**Comment by khrome83** [comment]
Yeah, it would be ideal. Added bonus -> green or red arrow for gain or loss? Or maybe positive vs negative percent? It could even just be a sentence.


"You have 21% more traffic than the previous timeframe."

But I think making it easy to scan would be ideal.

**Comment by adriaanvanrossum** [comment]
I think the colors are a great idea with arrow indicator.
